packages/daimo-api/src/api/getAccountHistory.ts

```diff
@@ -73,10 +75,10 @@ export async function getAccountHistory(
   }
 
   // Get the latest block + current balance.
-  const lastBlk = vc.getLastBlock();
```

Comment:
should we delete this function and associated code in `viemClient.ts`?

Reply:
I've been waiting to delete the unused ViemClient code until I get watcher 100% working